interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -1891,22 +1897,36 @@ private void assertServerStatsTrace(String method, Status.Code code,
     }
   }
 
-  private static void checkStartTags(MetricsRecord record, String methodName) {
+  private static void checkStartTags(MetricsRecord record, String methodName, boolean isClient) {
```

Comment:
Suggestion: Since there are many calls to checkStartTags and checkEndTags that are far from the definition, it might help to use an enum with CLIENT and SERVER values instead of a boolean.

Reply:
Make sense to me, though there's [other method](https://github.com/grpc/grpc-java/blob/master/interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java#L1948) in this class that already used a similar boolean check...